sparrow-blog-dao/src/main/java/com/sparrow/blog/dao/dos/PostsEntityDO.java

```diff
@@ -0,0 +1,143 @@
+package com.sparrow.blog.dao.dos;
+
+
+import com.baomidou.mybatisplus.annotation.IdType;
+import com.baomidou.mybatisplus.annotation.TableField;
+import com.baomidou.mybatisplus.annotation.TableId;
+import com.baomidou.mybatisplus.annotation.TableName;
+
+import java.util.Date;
+
+@TableName("posts")
+public class PostsEntityDO {
```

Comment:
I think there is a missing `@Data` annotation
```suggestion
@Data
@TableName("posts")
public class PostsEntityDO {
```

Reply:
I want not write getter setter method per field everyone.

Reply:
That's cool, how do others use the Class? I see all fields are private permission to access.